worldengine/draw.py

```diff
@@ -477,3 +554,8 @@ def draw_ancientmap_on_file(world, filename, resize_factor=1,
                     draw_biome, draw_rivers, draw_mountains, draw_outer_land_border, 
                     verbose)
     img.complete()
+
+def draw_scatter_plot_on_file(world, filename):
+    img = ImagePixelSetter(512, 512, filename)
+    draw_scatter_plot(world, 512, img)
```

Comment:
Yeah. That's a little bit of future proofing. While I think it would be a mistake to make it so that the size can be changed I didn't want to hardwire it in case I missed something and there was a compelling reason). 

This way if I'm wrong it requires much less in the way of code change.